### PR TITLE
Remove the class name "is-style-default"

### PR DESCRIPTION
**Pull request by noruzzamans**
@carolinan I have removed the 'is-style-default' class from some patterns and tested it. I did not find any design issues, and all patterns work perfectly as before. Please review and let me know if any further changes are needed. Thanks.

### CODE REVIEW

File: patterns/banner-with-description-and-images-grid.php [hunk 1]
```diff
@@ -41,4 +41,4 @@
 	</div>
 	<!-- /wp:group -->
 </div>
-<!-- /wp:group -->
```
Comment (juanfra) [inline patterns/banner-with-description-and-images-grid.php]: Please do not change these lines.

File: patterns/contact-location-and-link.php [hunk 2]
```diff
@@ -33,11 +33,11 @@
...
-			<!-- wp:image {"aspectRatio":"1","scale":"cover","linkDestination":"none","className":"wp-block-image size-large is-style-default"} -->
-			<figure class="wp-block-image size-large is-style-default"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/location.webp" alt="The business location" /></figure>
+			<!-- wp:image {"aspectRatio":"1","scale":"cover","linkDestination":"none","className":"wp-block-image size-large"} -->
+			<figure class="wp-block-image size-large"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/location.webp" alt="The business location" /></figure>
 			<!-- /wp:image --></div>
 		<!-- /wp:column -->
 		</div>
 	<!-- /wp:columns -->
 </div>
-<!-- /wp:group -->
```
Comment (juanfra) [inline patterns/contact-location-and-link.php]: Please do not change these lines.

File: patterns/more-posts.php [hunk 3]
```diff
@@ -31,4 +31,4 @@
 	</div>
 	<!-- /wp:query -->
 </div>
-<!-- /wp:group -->
+<!-- /wp:group -->
```
Comment (juanfra) [inline patterns/more-posts.php]: Please do not change these lines.

File: patterns/services-three-columns.php [hunk 4]
```diff
@@ -75,4 +75,4 @@
 	</div>
 	<!-- /wp:columns -->
 </div>
-<!-- /wp:group -->
+<!-- /wp:group -->
```
Comment (juanfra) [inline patterns/services-three-columns.php]: Please do not change these lines.

File: patterns/text-only-blog-single-post-template.php [hunk 5]
```diff
@@ -44,4 +44,4 @@
 </main>
 <!-- /wp:group -->
 
-<!-- wp:template-part {"slug":"footer","tagName":"footer"} /-->
+<!-- wp:template-part {"slug":"footer","tagName":"footer"} /-->
```
Comment (juanfra) [inline patterns/text-only-blog-single-post-template.php]: Please do not change these lines.

**Review by juanfra — COMMENTED** [review]
Thank you for working on this. It looks like you're modifying lines that are unrelated to the PR. Can you please fix that?

It seems that we have two PRs for this issue, the other one is [here](https://github.com/WordPress/twentytwentyfive/pull/251).

**Comment by noruzzamans** [comment]
@juanfra I've updated the file. Please review and let me know if any changes are needed. Thanks

**Comment by noruzzamans** [comment]
> Thank you for working on this. It looks like you're modifying lines that are unrelated to the PR. Can you please fix that?
> 
> It seems that we have two PRs for this issue, the other one is [here](https://github.com/WordPress/twentytwentyfive/pull/251).

@juanfra Yes, I have seen another PR with the same issues. I believe he submitted that PR after mine.

**Comment by juanfra** [comment]
Thanks @noruzzamans - I merged the other PR with credits for you as well, and all of the other participants [here](https://github.com/WordPress/twentytwentyfive/commit/883a7b2bb268f81da5494188827263a773360434).

I will close this one now. Thanks again!

**Pull request closed without merging**